libs/community/langchain_community/vectorstores/chroma.py

```diff
@@ -33,21 +33,34 @@
 DEFAULT_K = 4  # Number of Documents to return.
 
 
-def _results_to_docs(results: Any) -> List[Document]:
-    return [doc for doc, _ in _results_to_docs_and_scores(results)]
-
-
-def _results_to_docs_and_scores(results: Any) -> List[Tuple[Document, float]]:
-    return [
-        # TODO: Chroma can do batch querying,
-        # we shouldn't hard code to the 1st result
-        (Document(page_content=result[0], metadata=result[1] or {}), result[2])
-        for result in zip(
-            results["documents"][0],
-            results["metadatas"][0],
-            results["distances"][0],
-        )
-    ]
+def _results_to_docs(
+    results: Any, include_id: Optional[bool] = False
```

Comment:
**Static Code Issue**: Dynamically typed expressions (typing.Any) are disallowed in 'results' <br> **Fix**: Specify a more precise type instead of 'Any'. <br> **Code Suggestion**: 
 ```
 - results: Any, include_id: Optional[bool] = False
 + results: dict, include_id: Optional[bool] = False
 ```